docs/source/release/v6.6.0/diffraction.rst

```diff
@@ -10,34 +10,60 @@ Powder Diffraction
 
 New features
 ############
-.. amalgamate:: Diffraction/Powder/New_features
+- Introduce an input parameter in :ref:`WANDPowderReduction <algm-WANDPowderReduction>` to specify that the input workspaces are from event filtering. In such a situation, the overall reduction time could be reduced significantly.
+- Added support for ILL D4C instrument, a liquid diffractometer. This adds a new algorithm responsible for the reduction stage: :ref:`D4ILLReduction <algm-D4ILLReduction>`, which handles calling the loader, correcting for dead time, bank position offsets, relative efficiency, and normalisation to monitor or time.
+- Increase the speed of the fourier filter in the POLARIS total scattering reduction by reducing the Rmax parameter used in the pair of (forward and backward) PDFFourierTransform calls and optimising the integration code inside :ref:`PDFFourierTransform v2 <algm-PDFFourierTransform-v2>`
 
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Powder/Bugfixes
+- Change to use time average mean value of logs in :ref:`PDDetermineCharacterizations <algm-PDDetermineCharacterizations>`
+- Fix problem when calling focus twice in a reduction using the ISIS Powder diffraction scripts. The second run of focus was using a partially processed input file from the first focus resulting in some of the reduction steps (normalisation, absorption correction) were being run twice
+- Fixed an issue with :ref:`HB2AReduce <algm-HB2AReduce>` when 'colltrans' column is not available in the data file. This corresponds to the situation when the 'colltrans' motor is not connected.
 
 
 Engineering Diffraction
 -----------------------
 
 New features
 ############
-.. amalgamate:: Diffraction/Engineering/New_features
+- On the GSASII tab, when a successful refinement is run, the output table workspace for the lattice parameters now includes the Microstrain value and the column title is marked when this parameter has been refined. A similar instrument parameter table workspace is now also generated which includes Sigma-1 and Gamma(Y), with a refined flag, and the fitting range. The Pawley reflections are output to a separate table workspace if they are available.
+- An indeterminate progress bar has been added below the FitPropertyBrowser on the Engineering Diffraction Fitting tab. This will display when the fit is in progress and whether the recent fit status was success or fail.
 
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Engineering/Bugfixes
+- Correct the tabbing order between widgets on the Engineering Diffraction interface.
+- Add a scrollbar to the GSASII tab to allow a smaller interface height, like the fitting tab.
+- Check that there is enough data points before attempting to fit a diffraction peak in :ref:`FitPeaks <algm-FitPeaks>`
```

Comment:
My grammar could be at fault here, but I would have written this using "are" instead of "is":

> Check that there are enough data points...